erpnext/hr/doctype/attendance/attendance_calendar.js

```diff
@@ -15,4 +15,21 @@ frappe.views.calendar["Attendance"] = {
 		}
 	},
 	get_events_method: "erpnext.hr.doctype.attendance.attendance.get_events"
-};
\ No newline at end of file
+};
+
+frappe.views.calendar["AttendanceGantt"] = {
```

Comment:
AttendanceGantt is not a valid doctype. You should not create another configuration for calendar, just override the above one.